### DISCUSSION TITLE
DAG.owner = [] support array type

### DISCUSSION THREAD

**Discussion by seunggabi** [apache/airflow#27918] [General]
### if dag assigned owner=["a", "b"]: 
- not error
- dagbag not viewing..

### Use case/motivation

support multiple owner 
I will fix!

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
II am not sure if this issue is enough to describe what you want, but I think you should explain what you want to do and how you would represent multiple owners. Owner is a bit ugly child of Airflow, because it does not mean much. I think - either describe what you really want to achieve and how - or if you have a code ready to show - create a PR and explain what you want to do there. 

In any case thisi s a discussion - and you do not have to create feature to make a PR.
